src/odemis/acq/test/flim_test.py

```diff
@@ -54,6 +53,12 @@ class TestFlim(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        try:
+            # The nikonc driver only works for python2.
```

Comment:
No, that's not really true. Let's write "The nikonc driver needs omniorb which is not packaged in Ubuntu anymore".